daangn/minjee/real_estate.html

```diff
@@ -0,0 +1,436 @@
+                </svg>
+            </a>
+            <div class="toggle-btn">
+                <button class="navbar-toggler border-0 DoSearch-Button" type="button" data-bs-toggle="collapse" data-bs-target="#navbarRightContent" aria-controls="navbarRightContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20">
+                        <path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path>
+                    </svg>
+                </button>
+                <button class="navbar-toggler border-0" type="button" data-bs-toggle="collapse" data-bs-target="#navbarLeftContent" aria-controls="navbarLeftContent" aria-expanded="false" aria-label="Search">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            </div>
+            <div class="collapse navbar-collapse" id="navbarLeftContent">
+                <ul class="navbar-nav d-flex flex-wrap me-auto mb-2 mb-lg-0">
+                    <li class="nav-item me-2"><a class="nav-link fs-5 active" aria-current="page" href="main.html">중고거래</a></li>
+                    <li class="nav-item me-2"><a class="nav-link fs-5" href="nearby_store.html">동네가게</a></li>
+                    <li class="nav-item me-2"><a class="nav-link fs-5" href="alba.html">알바</a></li>
+                    <li class="nav-item me-2"><a class="nav-link fs-5" href="real_estate.html">부동산 직거래</a></li>
+                    <li class="nav-item me-2"><a class="nav-link fs-5" href="used_car.html">중고차 직거래</a></li>
+                </ul>
+            </div>
+            <input class="search-input form-contrl me-2 collapse navbar-collapse" id="navbarRightContent" type="search" size="28" placeholder="물품이나 동네를 검색해보세요" aria-label="Search">
+            <div class="d-none d-md-block">
+                <a href="https://chat.daangn.com/login" class="btn border text-black fw-bold">채팅하기</a>
+            </div>
+        </div>
+    </nav>
+    <main>
+        <section class="main__dsc pt-4 bg-real-estate">
+            <div class="row w-50 d-flex mx-auto">
+                <div class="col-12 col-md-5 my-auto mx-auto justify-content-center align-items-center">
+                    <h2 class="fw-bold mt-3">복비없이 투명한 <br> 부동산 직거래</h2>
+                    <p class="pb-5">이웃이 살던 집, 당근마켓에서<br>편하게 직거래해보세요.</p>
```

Comment:
```suggestion
                    <p class="pb-5 d-none d-md-block">이웃이 살던 집, 당근마켓에서<br>편하게 직거래해보세요.</p>
```

두 클래스를 추가하면 화면이 768px보다 작을때 p 태그를 숨길 수 있어요!